src/args.rs

```diff
@@ -10,42 +10,55 @@ use apca::api::v2::order;
 use chrono::NaiveDate;
 use chrono::NaiveDateTime;
 
-use num_decimal::Num;
+use clap::Args as ClapArgs;
+use clap::ArgEnum;
+use clap::ArgGroup;
+use clap::Parser;
+use clap::Subcommand;
 
-use structopt::clap::ArgGroup;
-use structopt::StructOpt;
+use num_decimal::Num;
 
 use uuid::Error as UuidError;
 use uuid::Uuid;
 
 
 /// A command line client for automated trading with Alpaca.
-#[derive(Debug, StructOpt)]
-#[structopt(about, version = env!("VERSION"))]
+#[derive(Debug, Parser)]
+#[clap(name = "apcacli")]
+#[clap(author = "Daniel Mueller <deso@posteo.net>")]
```

Comment:
I'd suggest we leave the author out of the help text. It doesn't look so great there, in my opinion.

Reply:
Done.